selfdrive/controls/radard.py

```diff
@@ -166,7 +166,7 @@ def update(self, frame, sm, rr, has_radar):
     dat.radarState.radarErrors = list(rr.errors)
     dat.radarState.controlsStateMonoTime = sm.logMonoTime['controlsState']
 
-    if has_radar:
+    if enable_lead:
```

Comment:
I think we want the radar data in the logs even if we're not going to show the lead

Reply:
since leads don't depend on having a radar at all, would you like `leadOne`/`leadTwo` always in the logs and use something new to determine if leads are displayed in UI?

Reply:
added `radarOffCan` back in as an `or` condition to preserve existing behavior (even if wrong)

Reply:
แก้ปัญหาโดย เตชสิทธ์ ล้วนทร